src/api/status/src/js/feed-stats.js

```diff
@@ -1,11 +1,15 @@
 const { fetch } = require('@senecacdot/satellite');
 
+const { POSTS_URL, POSTS_PORT } = process.env;
+
 module.exports = async function getFeedCount() {
   let feedCount = 0;
   try {
-    const data = await fetch(`${process.env.API_URL}/v1/posts/feeds`, {
-      method: 'HEAD',
-    });
+    const API_URL = POSTS_URL
```

Comment:
I don't think you should redefine what `API_URL` means, since we use it everywhere to mean something different than this.

Reply:
Noted, will change to another variable names